src/char/char_mapif.cpp

```diff
@@ -1132,10 +1133,10 @@ int chmapif_parse_updfamelist(int fd){
 
             switch(type)
             {
-				case RANK_BLACKSMITH:	size = fame_list_size_smith;	list = smith_fame_list;		break;
-				case RANK_ALCHEMIST:	size = fame_list_size_chemist;	list = chemist_fame_list;	break;
-				case RANK_TAEKWON:		size = fame_list_size_taekwon;	list = taekwon_fame_list;	break;
-				default:				size = 0;						list = NULL;				break;
+            case RANK_BLACKSMITH:	size = fame_list_size_smith;	list = smith_fame_list;		break;
+            case RANK_ALCHEMIST:	size = fame_list_size_chemist;	list = chemist_fame_list;	break;
+            case RANK_TAEKWON:		size = fame_list_size_taekwon;	list = taekwon_fame_list;	break;
+            default:				size = 0;						list = NULL;				break;
```

Comment:
Style change only? Could be reverted for now?

Reply:
The whole function is mixed tab and space indentation. Anyway I'll revert it. We should keep style changes in explicit commits.